modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -108,23 +121,32 @@ curl "http://localhost/myResource" -u 1234:abcd
 
 Example DataWeave 2.0 expression to be used when configuring the policy:
 
-No DataWeave 2.0 expression needed to be specified when the policy is configured to use the Basic Authentication as the origin of the credentials.
+To use the Basic Authentication as the origin of the credentials, no DataWeave 2.0 expression needs to be specified when configuring the policy.
```

Comment:
This is a bit confusing, no expression needs to be configured because they have to select the other option (not custom headers).

Reply:
This was there from before. I thought this is a requirement. If it does not make sense, should I remove it?

Reply:
Done! Thanks!